main.tf

```diff
@@ -289,12 +289,12 @@ resource "google_compute_firewall" "iap-tcp-forwarding" {
   }
 
   source_ranges = var.allowed_ssh_ranges
-  target_tags   = ["iap-tcp-forwarding-allowed"]
+  target_tags   = ["iap_tcp_forwarding-allowed"]
```

Comment:
Was this caught by linting too? I feel like we should decide on a tag format (dashes or underscore) independent of the terraform resource standard

Reply:
No, this was unintentional.